### PR TITLE
add EOA based paymaster

### PR DESCRIPTION
**Pull request by unclezoro**

### CODE REVIEW

File: docs/bnb-smart-chain/developers/paymaster/paymaster-api.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+```plain
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "pm_isSponsorable",
+  "params": [
+    {
+      "to": "0x...", // an address
+      "from": "0x...", // an address"value": "0xa1",
+      "data": "0x",
```
Comment (clydemeng) [inline docs/bnb-smart-chain/developers/paymaster/paymaster-api.md:38]: shall we add "value" param in the example?

File: mkdocs.yml [hunk 2]
```diff
@@ -123,6 +123,10 @@ nav:
             - Docker Image: ./bnb-smart-chain/developers/node_operators/docker.md
             - Upgrade Geth: ./bnb-smart-chain/developers/node_operators/upgrade_geth.md
             - Node Maintenance: ./bnb-smart-chain/developers/node_operators/node_maintenance.md
+        - EOA Paymaster:
+            - Overview: ./bnb-smart-chain/developers/paymaster/overview.md
+            - Paymaster API Spec: ./bnb-smart-chain/developers/paymaster/paymaster-api.md
+            - Wallet Integration: ./bnb-smart-chain/developers/paymaster/wallet-integrate.md
```
Comment (clydemeng) [inline mkdocs.yml:129]: the file name is conventionally a noun, while "integrate" is a verb.

File: docs/bnb-smart-chain/developers/paymaster/paymaster-api.md (hunk 1 above)
Comment (unclezoro) [reply docs/bnb-smart-chain/developers/paymaster/paymaster-api.md:38]: fixed.

File: mkdocs.yml (hunk 2 above)
Comment (unclezoro) [reply mkdocs.yml]: fixed

**Review by clydemeng — APPROVED** [review]

**Review by ruojunm — APPROVED** [review]

**Comment by Bassambmbmbm** [comment]
ياعم زورو ...؟ اريد ان تعطيني بعض المعلومات

**Pull request merged**